Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/orientation.h

```diff
@@ -1364,6 +1364,9 @@ void orient_to_bound_a_volume(TriangleMesh& tm,
   typedef typename GetGeomTraits<TriangleMesh, NamedParameters>::type              GT;
   typedef typename GetInitializedFaceIndexMap<TriangleMesh, NamedParameters>::type FaceIndexMap;
 
+  std::size_t nbf = num_faces(tm);
+  if (nbf == 0) return;
```

Comment:
[`is_empty(g)`](https://github.com/CGAL/cgal/blob/master/BGL/include/CGAL/boost/graph/helpers.h#L994)?

Reply:
`num_faces()` may even be larger than zero if there is garbage.